static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -184,11 +185,6 @@
     }
 }
 
-.content__main-column--gallery {
-    margin-right: auto;
-    padding-bottom: $gs-baseline*2;
-}
-
```

Comment:
Might be worth checking the style changes in this PR don't have a negative effect on old galleries.